specificatie/gba-genereervariant/openapi.json

```diff
@@ -698,31 +392,85 @@
             "pattern" : "^[a-zA-Z0-9\\.,_ ]{1,80}$",
             "type" : "string",
             "description" : "Beschrijving van de fout op de parameterwaarde",
-            "example" : "Waarde is geen geldige integer."
+            "example" : "Waarde is geen geldig getal."
           }
         },
         "description" : "Details over fouten in opgegeven parameters"
       },
-      "ReisdocumentenQuery" : {
-        "required" : [ "burgerservicenummer" ],
+      "GbaReisdocument" : {
         "type" : "object",
         "properties" : {
-          "burgerservicenummer" : {
-            "$ref" : "#/components/schemas/Burgerservicenummer"
+          "reisdocumentnummer" : {
+            "$ref" : "#/components/schemas/Reisdocumentnummer"
+          },
+          "soort" : {
+            "$ref" : "#/components/schemas/Waardetabel"
+          },
+          "datumEindeGeldigheid" : {
+            "$ref" : "#/components/schemas/GbaDatum"
+          },
+          "inhoudingOfVermissing" : {
+            "$ref" : "#/components/schemas/GbaInhoudingOfVermissing"
+          },
+          "houder" : {
+            "$ref" : "#/components/schemas/GbaReisdocumenthouder"
+          },
+          "inOnderzoek" : {
+            "$ref" : "#/components/schemas/GbaInOnderzoek"
           }
         }
       },
-      "GbaReisdocumentenResponse" : {
+      "Reisdocumentnummer" : {
+        "pattern" : "^[A-Z0-9]{9}$",
+        "type" : "string"
+      },
+      "GbaInhoudingOfVermissing" : {
         "type" : "object",
         "properties" : {
-          "reisdocumenten" : {
-            "type" : "array",
-            "items" : {
-              "$ref" : "#/components/schemas/GbaReisdocument"
+          "datum" : {
+            "$ref" : "#/components/schemas/GbaDatum"
+          },
+          "aanduiding" : {
+            "$ref" : "#/components/schemas/Waardetabel"
+          }
+        }
+      },
+      "GbaReisdocumenthouder" : {
```

Comment:
informatievraag? Misschien hernoemen naar persoon en in de proxy documenthouder introduceren?

Reply:
Ik begrijp nu dat dit onder "structureren" valt.

Reply:
ik denk dat dit geen informatievraag is. gegevens worden niet bewerkt